docs/understanding-serverless/serverless-yml.md

```diff
@@ -18,16 +18,14 @@ provider:
     subnetIds:
       - subnetId1
       - subnetId2
+  stage: beta # Overwrite the default "dev" stage.
```

Comment:
yep...the default config in `default` and `provider` objects are an exact reflection of each other for backward compatibility, so aws deployment always have access to those values.